modules/js/planetaryFinal.js

```diff
@@ -76,129 +101,114 @@ function addPlanetaryGearComposite(centerX, centerY, constraintX, constraintY){
-  addPlanetaryGearComposite(compositeArray[0].constraints[0].pointA.x - compositeArray[0].radius - radius - ((toothHeight*0.6)*2), (window.innerHeight)*(0.65),compositeArray[0].constraints[0].pointA.x,compositeArray[0].constraints[0].pointA.y);
-}
-// see smallGear1()
-function mediumGear1(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 64;
-  compositeArray[1].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 3;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  removeComposite(compositeArray[1].bodies[0])
-  addPlanetaryGearComposite(compositeArray[0].constraints[0].pointA.x - compositeArray[0].radius - radius - ((toothHeight*0.6)*2), (window.innerHeight)*(0.65),compositeArray[0].constraints[0].pointA.x,compositeArray[0].constraints[0].pointA.y);
+  addPlanetaryGearComposite(
+    compositeArray[0].constraints[0].pointA.x -
+      compositeArray[0].radius -
+      radius -
+      toothHeight * 0.6 * 2,
+    window.innerHeight * 0.65,
+    compositeArray[0].constraints[0].pointA.x,
+    compositeArray[0].constraints[0].pointA.y
+  );
 }
-// see smallGear1()
-function largeGear1(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 80;
-  compositeArray[1].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 2;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  removeComposite(compositeArray[1].bodies[0])
-  addPlanetaryGearComposite(compositeArray[0].constraints[0].pointA.x - compositeArray[0].radius - radius - ((toothHeight*0.6)*2), (window.innerHeight)*(0.65),compositeArray[0].constraints[0].pointA.x,compositeArray[0].constraints[0].pointA.y);
-}
-// see smallGear1()
-function smallGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 48;
-  compositeArray[0].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 4;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  changeBodyPlanetary(0);
-}
-// see smallGear1()
-function mediumGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 64;
-  compositeArray[0].radius = radius
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 3;
-  toothWidth = (toothWidthDegree/conversionFactor);
-  changeBodyPlanetary(0);
-}
-// see smallGear1()
-function largeGear2(){
-  Body.setAngle(compositeArray[0].bodies[0], 0)
-  Body.setAngle(compositeArray[1].bodies[0], 0)
-  radius = 80;
+
+function gear2(rad) {
+  Body.setAngle(compositeArray[0].bodies[0], 0);
+  Body.setAngle(compositeArray[1].bodies[0], 0);
+  radius = rad;
   compositeArray[0].radius = radius;
-  steps = (0.25 * radius)*2;
-  toothWidthDegree = 2;
-  toothWidth = (toothWidthDegree/conversionFactor);
+  steps = 0.25 * radius * 2;
+  toothWidthDegree = 4;
+  toothWidth = toothWidthDegree / conversionFactor;
```

Comment:
Are these local variables?  I don't see a declaration.  We probably want to avoid global variables if possible and keep these as `let` or `const` if they are truly local.